retriever/engines/mysql.py

```diff
@@ -39,6 +43,26 @@ class engine(Engine):
                       "{db}.{table}"),
                      ]
 
+    def check_credentials(self):
+        self.get_input()
+        try:
+            sql_conf = open_fr(MYSQL_CONF_PATH)
+            sql_conf_lines = sql_conf.readlines()
+            if self.opts["user"] != "root":
+                for line in sql_conf_lines:
```

Comment:
@henrykironde the file path `~/.my.cnf` is for user specific options. And global options are usually set by the user in `/etc/mysql/my.cnf` usually and the users to store their root credentials in the file path. That is why I excluded user="root" as it was searching for `~/.my.cnf`. Shall I change the file path to search in for root user?